### ISSUE TITLE
can not reset oldCell.label.textColor, two or more items  be selected

### ISSUE THREAD

**Issue by KeGao** [xmartlabs/XLPagerTabStrip#263]
![img_0173](https://cloud.githubusercontent.com/assets/10496994/19726466/9d66f344-9bbd-11e6-959a-22c7abe60f73.PNG)
![img_0174](https://cloud.githubusercontent.com/assets/10496994/19726478/abe061da-9bbd-11e6-9850-4a22d1ab0ea7.PNG)

特殊情况下回出现这个问题 

my code:
self.changeCurrentIndexProgressiveBlock = ^void(XLButtonBarViewCell *oldCell, XLButtonBarViewCell *newCell, CGFloat progressPercentage, BOOL changeCurrentIndex, BOOL animated){
        if (changeCurrentIndex) {
            [oldCell.label setTextColor:[UIColor colorWithWhite:0.000 alpha:0.400]];
            [newCell.label setTextColor:[Constant SharedConstant].colors[0]];  //设置字体颜色

```
        if (animated) {
            [UIView animateWithDuration:0.1
                             animations:^(){
                                 newCell.transform = CGAffineTransformMakeScale(1.0, 1.0);
                                 oldCell.transform = CGAffineTransformMakeScale(0.8, 0.8);
                             }
                             completion:nil];
        }
        else{
            newCell.transform = CGAffineTransformMakeScale(1.0, 1.0);
            oldCell.transform = CGAffineTransformMakeScale(0.8, 0.8);
        }
    }
};
```

**Comment by 74981** [comment]
In ButtonBarPagerTabsStripViewController.swift at method ScrollViewDidEndScrollingAnimation-

Add following line- self.currentIndexPath = nil to make currentIndex nil before scrolling.

**Comment by m-revetria** [comment]
Hi @KeGao, this issue looks related to #338 which has been recently solved. Please check if that solves your issue.

**Comment by ivanmih** [comment]
Hey @m-revetria i am author of #338 and the problem still remains. 
Aside of the problem, i even had several crashes if i implement new code. The crash is on this line:
` if !indexPathsToReload.isEmpty {
     buttonBarView.reloadItems(at: indexPathsToReload)
 }`

**Comment by santiagofm** [comment]
Hey @ivanmih, are you pointing your podfile/carthage to the master branch ? We haven't released a new version that includes the fix yet.
As for the crashes, if it's not related to this issue please create a new one with more details/sample code.

Cheers

**Comment by ivanmih** [comment]
@santiagofm actually i changed those lines directly in ButtonBarPagerTabStripViewController file in my Pods directory. I will open new issue, although it is related to this one.

**Comment by santiagofm** [comment]
@ivanmih Could you provide some sample project to reproduce the issue? It was reproducible in the Example project but that was fixed in 4c609a92b17d4f9e2171adf980f00d3d01656250 and b77a9a72ce2b125210c65e6c52451272a939f315.

Cheers
